Dockerfile.format

```diff
@@ -0,0 +1,35 @@
+#
```

Comment:
Maybe we could move this into `build-support`. That would also help to not recreate the image each time any cpp file is changed.

Reply:
Moved. But I don't get why will the image be recreated each time any cpp file is changed.